tests/quickcheck_test/test_quickcheck.py

```diff
@@ -0,0 +1,52 @@
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+"""This module tests the CodeChecker quickcheck feature."""
+
+import glob
+import os
+import subprocess
+import unittest
+
+
+class QuickCheckTestCase(unittest.TestCase):
+    """This class tests the CodeChecker quickcheck feature."""
+
+    @classmethod
+    def setup_class(cls):
+        """Setup the class."""
+        cls.package_dir = \
+            os.path.realpath(os.environ.get('TEST_CODECHECKER_DIR'))
+
+        # Put CodeChecker/bin to PATH so CodeChecker command becomes available.
+        cls.env = os.environ

```

Comment:
use os.environ.copy() to create a new environment which is used only for the current tests, without the copy it will modify the running process environment